### PR TITLE
Fix help indentation for starting url.

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const cli = meow(`
   	--port: emote debugging port number to use.
```
Comment (paulirish) [inline src/cli.ts:10]: s/emote/Remote/

File: src/cli.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ const cli = meow(`
...
   	--chrome-path: Explicit path of intended Chrome binary
   	--user-data-dir: Chrome profile path to use
-		--starting-url: Starting URL to open the browser with, default is about:blank
+	--starting-url: Starting URL to open the browser with, default is about:blank
 	Chrome Flags
 		Additional flags to pass to Chrome like '--headless', '--disable-gpu'
 	Examples
```
Comment (paulirish) [inline src/cli.ts:16]: and down on L20 i dont think "chrome-launcher --headless" actually would work if i'm reading this right.

**Review by paulirish — CHANGES_REQUESTED** [review]

File: src/cli.ts (hunk 2 above)
Comment (paulirish) [reply src/cli.ts:16]: ok ill do these :)

**Comment by ragingwind** [comment]
Thanks.

File: src/cli.ts (hunk 2 above)
Comment (ragingwind) [reply src/cli.ts:16]: @paulirish I'll fix it. thanks! *update* done by https://github.com/ragingwind/chrome-launcher-cli/commit/cea05a2c86ab8adf4fc2e3b196d27f7f587fc725

**Pull request merged**